src/main/java/com/example/onlinebookshop/controller/BookController.java

```diff
@@ -30,23 +32,24 @@
 @Validated
 @RequiredArgsConstructor
 @RestController
-@Tag(name = Constants.BOOK_API_NAME,
-        description = Constants.BOOK_API_DESCRIPTION)
+@Tag(name = BookConstants.BOOK_API_NAME,
```

Comment:
If you use constant, please use import static 
```suggestion
@Tag(name = BOOK_API_NAME,
```

Check all places